root/etc/cont-init.d/12-samples

```diff
@@ -1,21 +1,34 @@
 #!/usr/bin/with-contenv bash
 
 # remove old samples
-find /config/nginx/ -name "*.conf.sample" -type f -delete
+find /config/nginx/ \
+    -name "*.conf.sample" \
+    -type f \
+    -delete
 
-# copy new samples
-cp \
-    /defaults/nginx/*.conf.sample \
-    /config/nginx/
-cp \
-    /defaults/nginx/http-confs/*.conf.sample \
-    /config/nginx/http-confs/
-cp \
-    /defaults/nginx/location-confs/*.conf.sample \
-    /config/nginx/location-confs/
-cp \
-    /defaults/nginx/server-confs/*.conf.sample \
-    /config/nginx/server-confs/
-cp \
-    /defaults/nginx/site-confs/*.conf.sample \
-    /config/nginx/site-confs/
+# copy new samples
```

Comment:
WDYT about doing `cp -fR /defaults/nginx/samples/ /config/nginx/`, and put all the samples in that folder in the correct structure?

Reply:
https://discord.com/channels/354974912613449730/512708069709316137/946789530608992256 (internal discussion)